stripe/api_resources/error_object.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import absolute_import, division, print_function
+
+from stripe.util import merge_dicts
+from stripe.stripe_object import StripeObject
+
+
+class ErrorObject(StripeObject):
+    def refresh_from(
+        self,
+        values,
+        api_key=None,
+        partial=False,
+        stripe_version=None,
+        stripe_account=None,
+        last_response=None,
+    ):
+        # Unlike most other API resources, the API will omit attributes in
+        # error objects when they have a null value. We manually set default
+        # values here to facilitate generic error handling.
+        values = merge_dicts(
+            {
+                "charge": None,
+                "code": None,
+                "decline_code": None,
+                "doc_url": None,
+                "message": None,
+                "param": None,
+                "payment_intent": None,
+                "payment_method": None,
+                "setup_intent": None,
+                "source": None,
+                "type": None,
```

Comment:
Do we do the same in ruby? That feels like something we'll never remember to fill out unless it could be autogenerated from the spec

Reply:
We do something different but with the same goal in mind in stripe-ruby: https://github.com/stripe/stripe-ruby/blob/3cb9ad7fca5bfd372006d551d1ecc1089d3739aa/lib/stripe/error_object.rb#L8-L77

That approach wouldn't work in stripe-python because accessing a key that does not exist in Ruby returns `nil`, but in Python it raises a `KeyError`.

The approach here is the same one as in stripe-php: https://github.com/stripe/stripe-php/blob/9a58f033d9d126c3785f0d8ad47f2e1db269864a/lib/ErrorObject.php#L144-L160.

We can look at autogenerating these classes in all languages if maintenance becomes an issue.

Reply:
Ack. I just worry we will never remember this exists and the `error` object is being changed liberally lately.